identifier/identifier.go

```diff
@@ -80,13 +93,28 @@ func NewDNS(domain string) ACMEIdentifier {
 // NewDNSSlice is a convenience function for creating a slice of ACMEIdentifier
 // with Type "dns" for a given slice of domain names.
 func NewDNSSlice(input []string) ACMEIdentifiers {
-	var out []ACMEIdentifier
+	var out ACMEIdentifiers
 	for _, in := range input {
 		out = append(out, NewDNS(in))
 	}
 	return out
 }
 
+// ToDNSSlice returns a list of DNS names from the input, if the input contains
```

Comment:
Optional: I don't think a comma is necessary here.

```suggestion
// ToDNSSlice returns a list of DNS names from the input if the input contains
```